package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "klaudsol-cms",
-  "version": "3.5.0",
+  "version": "3.5.1",
```

Custom attribute types is a big deal, so this merits a minor change.

```suggestion
  "version": "3.6.0",
```